scripts/S5_defineGaps.R

```diff
@@ -54,6 +54,29 @@ crs(vecRemove) <- "epsg: 32617"
 writePath <- "droneData/droneOrthomosaics/shapefiles/anomalyPolygons/structural/"
 writeVector(vecRemove, paste0(writePath,"anom_2023-10-12.shp"), overwrite=TRUE)
 
+#########################################################################################
+# Option 1: gdal
+## - spectral = ~1 minute
+## - structural = 5 seconds
+
+## ------------------------------------------------- ##
+# C. Identify gaps, calculate metrics, and save outputs
+gdalOutDir <- ifelse(changeType=="ortho", "intermediateIndex", "intermediateCanopy")
+gdalOutDirFull <- paste0("droneData/processedChange/", gdalOutDir)
+if(!dir.exists(gdalOutDirFull)) dir.create(gdalOutDirFull)
+
+saveGapFiles <- FALSE
+saveGapsPath <- "droneData/processedChange/gapsIndex/fileType/gapsD1_D2_res20.ext"
+

```

Comment:
sorry, if you're running this there should be this added: (when everything can be merged I'll go back and fix things)

```
saveGapsPath <- ifelse(changeType=="structural", 
                        gsub("gapsIndex", "gapsCanopy", saveGapsPath),
                        saveGapsPath)
```